lib/ddtrace/workers/queue.rb

```diff
@@ -26,6 +26,7 @@ def dequeue
         buffer.shift
       end
 
+      # Are there more items to be processed next?
       def work_pending?
         !buffer.empty?
       end
```

Comment:
I'm a big fan of self-describing code. If we like the comment, why not turning this into `def more_items_to_be_proccessed?` or similar? :)

Reply:
I do agree that comments <<< well-named-things.

I mean, isn't "more_items_to_be_processed?" =~ "work_pending?" 😄 I think the main issue is that, like all things complex, there's nuance that would be hard to capture in the name.

Maybe "more_items_to_be_processed?" or similar would be better, but I'm _sneaking away with the ✅ for now for the non-flaky CI._